dev/dev-env/create.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2016
+set -euo pipefail
+
+destDir=${1:-nix-bitcoin}
```

Comment:
If we want to change anything about the dev-env (add, delete or update files), how does a dev with an existing dev-env upgrade it to the latest version?

Reply:
The user can re-run the `create.sh` script. No harm is done when user modifications are overwritten, because the dev env is under version control.
To allow future automatic updates, I've added file `lib/dev-env-version`.

Reply:
That the dev env is under version control is a good point. Also, good idea to add the `dev-env-version`.